db/index.js

```diff
@@ -140,9 +140,26 @@ const loadAll = (modelType = 'student') => {
       return data;
     })
     .catch((e) => {
-      console.log('\nError in \'loadAll\'');
+      console.log('\nError in \'loadAll\'', e);
+    });
+};
+
+// ========================
+// === Update Functions ===
+// ========================
+
+// hostType is the thing that the comment is on (ex. student, event, etc.)
+//   It is hostType instead of modelType, because it a part of the model, not the model itself
+// identifier is an object that is used to select the particular student, same as for loadParticular
+// comment is the comment object, it will be pushed onto the array of comments for the identified student
+const updateComments = (hostType = 'student', identifier, comment) => {
+  return getModelType(hostType)
+    .findOneAndUpdate(identifier, { $push: { 'comments': comment } }, { 'new': true })
+    .then((arg) => {
```

Comment:
Why the name 'arg'? This is the entire updated set of comments, right?

Reply:
Yeah, sometimes I just write 'arg' for the argument name. I should be more descriptive.

Reply:
Or is this your go-to generic identifier for promises? Like "data"

Reply:
I mostly have shifted to 'data' as my default, but 'arg' was my original, and sometimes it slips in